lib/src/backend/declarer.dart

```diff
@@ -84,11 +87,15 @@ class Declarer {
   /// If [collectTraces] is `true`, this will set [GroupEntry.trace] for all
   /// entries built by the declarer. Note that this can be noticeably slow when
   /// thousands of tests are being declared (see #457).
-  Declarer({Metadata metadata, bool collectTraces: false})
-      : this._(null, null, metadata ?? new Metadata(), collectTraces, null);
+  ///
+  /// If [doRetry] is `true`, this will retry failed tests if they are
```

Comment:
I think it would be clearer to describe the behavior if it's `false` first, since that's what users would actually set it to.